src/pages/docs/basics/index.mdx

```diff
@@ -65,6 +65,7 @@ The REST interface communicates with Ably using the HTTP protocol and is effecti
 * issue tokens on behalf of other realtime clients
 * retrieve persisted messages, presence history and application usage statistics
 
+
```

Comment:
We don't tend to add a 2nd line above a header. But you've done this in the PR. Is this something we should propose as a styling rule?